target/bin/delmailuser

```diff
@@ -91,9 +91,6 @@ for EMAIL in "${@}"
 do
   ERROR=false
 
-  # very simple plausibility check
-  [[ ${EMAIL} != *@*.* ]] && errex "No valid email address: ${EMAIL}"
```

Comment:
Could we instead mimic what addmailuser does so that there is still some validation in this file?

Reply:
I already wanted to agree, but then, think about it: If you were able to add addresses, you should be able to remove them, and restrictions would only hinder you here. We have a "closed context" in this case, i.e. we only remove lines in one file.

I'm fine with it either way though:)

Reply:
I don't fully get the script below here, is there currently a warning if the mailbox couldn't be found / doesn't exist? If not, I think that should be added, and if so I think this suggestion is covered by that warning?

Reply:
> is there currently a warning if the mailbox couldn't be found / doesn't exist? 

There should be a warning, however the check is currently broken. I've already fixed it locally and will provide a PR once this got merged.

Reply:
@NorseGaud are you fine with this change anyway?

Reply:
In that case I'm resolving this conversation, as that warning will inform users. I'll request a re-review from @NorseGaud.

Reply:
@georglauterbach go ahead